### PR TITLE
[BEARAXE] Fight Pit Bears now scale to server population.

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
Fight pit bears (the ones on Mother Russia Bleeds) now scale to player pop, maintaining their current stats at a player count of 20, peaking at 3.5x their current stats at 70 pop. Because of these changes, they are no longer gold core spawnable.

## Why It's Good For The Game
The bears will no longer slaughter on lowpop, nor will they be complete pushovers on highpop.

## Changelog
:cl:
balance: Fight Pit Bears now scale to player population.
/:cl:

### CODE REVIEW

File: code/modules/shuttle/special.dm [hunk 1]
```diff
@@ -310,6 +310,14 @@
 	name = "fight pit bear"
 	desc = "This bear's trained through ancient Russian secrets to fear the walls of its glass prison."
 	environment_smash = ENVIRONMENT_SMASH_NONE
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/bear/fightpit/Initialize()
+	. = ..()
+	var/M = round(GLOB.player_list.len / 20, 0.1)
```
Comment (Fikou) [inline code/modules/shuttle/special.dm]: you should use descriptive var names

File: code/modules/shuttle/special.dm [hunk 2]
```diff
@@ -310,6 +310,14 @@
...
+	src.maxHealth *= M
+	src.health *= M
+	src.melee_damage *= M
```
Comment (Fikou) [inline code/modules/shuttle/special.dm]: src is implied, don't do src.var

File: code/modules/shuttle/special.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/shuttle/special.dm]: also should use length(SSticker.mode.current_players[CURRENT_LIVING_PLAYERS]) and 20 should be a define probably

**Review by Fikou — DISMISSED** [review]

**Comment by ACanOpener** [comment]
good pr

**Review by PowerfulBacon — DISMISSED** [review]

File: code/modules/shuttle/special.dm [hunk 3]
```diff
@@ -310,6 +312,14 @@
...
 	desc = "This bear's trained through ancient Russian secrets to fear the walls of its glass prison."
 	environment_smash = ENVIRONMENT_SMASH_NONE
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/bear/fightpit/Initialize()
+	. = ..()
+	var/multiplier = round(length(SSticker.mode.current_players[CURRENT_LIVING_PLAYERS]) / BASE_BEAR_DIVISOR, 0.1)
+	maxHealth *= multiplier
+	health *= multiplier
+	melee_damage *= multiplier
```
Comment (PowerfulBacon) [inline code/modules/shuttle/special.dm]: Can you make sure its always greater than 0?

File: code/modules/shuttle/special.dm [hunk 4]
```diff
@@ -310,6 +312,14 @@
 	name = "fight pit bear"
 	desc = "This bear's trained through ancient Russian secrets to fear the walls of its glass prison."
 	environment_smash = ENVIRONMENT_SMASH_NONE
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/bear/fightpit/Initialize()
+	. = ..()
+	var/multiplier = round(length(SSticker.mode.current_players[CURRENT_LIVING_PLAYERS]) / BASE_BEAR_DIVISOR, 0.1)
```
Comment (ike709) [inline code/modules/shuttle/special.dm]:
```suggestion
	var/multiplier = max(round(length(SSticker.mode.current_players[CURRENT_LIVING_PLAYERS]) / BASE_BEAR_DIVISOR, 0.1), 1)
```

File: code/modules/shuttle/special.dm (hunk 3 above)
Comment (PowerfulBacon) [reply code/modules/shuttle/special.dm]: Also if theres like 5 people on the bears will be 1 hit kills basically

Comment (ike709) [reply code/modules/shuttle/special.dm]: Addressed in my review.

**Review by Archanial — APPROVED** [review]

**Pull request merged**